hue/templates/configmap.yaml

```diff
@@ -191,15 +138,15 @@ data:
       livy_server_session_kind={{ .Values.config.spark.livyServerSessionKind }}
 
       # Whether Livy requires client to perform Kerberos authentication.
-      security_enabled={{ .Values.config.spark.securityEnabled }}
+      #security_enabled={{ .Values.config.spark.securityEnabled }}
```

Comment:
Why do you comment this? Shouldn't be enough setting this to `false` or something like that?

Reply:
Fair point. I was strictly replicating the test config but it's better to set it to false.